src/components/ScholarshipForm.jsx

```diff
@@ -78,56 +90,111 @@ function ScholarshipForm({ setSubmissionAlert }) {
         <FormikTextField
           label="Scholarship Link *"
           id="website"
           formik={formik}
+          labelStyle={classes.inputLabel}
         />
-      </div>
+      </Grid>
+
+      <Grid item sm={6}>
+        <DatePicker
+          id="deadline"
+          label="Deadline *"
+          labelStyle={classes.inputLabel}
+          error={formik.touched.deadline && Boolean(formik.errors.deadline)}
+          helperText={formik.touched.deadline && formik.errors.deadline}
+          value={formik.values.deadline}
+          onChange={(date) => formik.setFieldValue('deadline', date, true)}
+        />
+      </Grid>
+
+      <Grid item>
+        <ScholarshipAmountField
+          helperText={
+            (getIn(formik.touched, 'amount.type') &&
+              getIn(formik.errors, 'amount.type')) ||
+            getIn(formik.errors, 'amount.min') ||
+            getIn(formik.errors, 'amount.max')
+          }
+          amountType={formik.values.amount.type ?? 'UNKNOWN'}
+          minAmount={formik.values.amount.min}
+          maxAmount={formik.values.amount.max}
+          onTypeChange={(e) =>
+            formik.setFieldValue('amount.type', e.target.value, true)
+          }
+          updateAmount={updateAmount}
+        />
+      </Grid>
 
-      <ScholarshipAmountField
-        className={classes.fieldStyle}
-        helperText={
-          (getIn(formik.touched, 'amount.type') &&
-            getIn(formik.errors, 'amount.type')) ||
-          getIn(formik.errors, 'amount.min') ||
-          getIn(formik.errors, 'amount.max') ||
-          ''
-        }
-        amountType={formik.values.amount.type}
-        minAmount={formik.values.amount.min}
-        maxAmount={formik.values.amount.max}
-        onTypeChange={(e) =>
-          formik.setFieldValue('amount.type', e.target.value, true)
-        }
-        updateAmount={updateAmount}
-      />
+      <Grid item xs={12}>
+        <FormikTextField
+          label="Description *"
+          id="description"
+          labelStyle={classes.inputLabel}
+          formik={formik}
+          rows={8}
+        />
+      </Grid>
+    </Grid>
+  );
+
+  stepperItems.Requirements = 'requirements';
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <Stepper activeStep={activeStep} orientation="vertical">
+        {Object.keys(stepperItems).map((key) => (
+          <Step key={key}>
+            <StepLabel>{key}</StepLabel>
+            <StepContent>
+              {stepperItems[key]}
+              <div className={classes.stepperBtns}>
+                <Button
+                  disabled={activeStep === 0}
+                  onClick={() => setActiveStep((prevStep) => prevStep - 1)}>
+                  BACK
+                </Button>
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => setActiveStep((prevStep) => prevStep + 1)}>
+                  NEXT
+                </Button>
```

Comment:
nit: I suggest making this the **SUBMIT** button when `activeStep == Object.keys(stepperItems).length` (or `key == 'Confirmation'`) so that it isn't possible to submit the form until the last step is shown.

Reply:
Nice, for now there is no back button when the submit button appears but I will fix this in another PR when adding the Review stage.